tests/devices/test_default_qubit_torch.py

```diff
@@ -914,6 +914,7 @@ def test_three_qubit_no_parameters_broadcasted(
         expected = qml.math.einsum("ij,lj->li", op_mat, state)
         assert torch.allclose(res, expected, atol=tol, rtol=0)
 
+    @pytest.mark.usefixtures("use_new_opmath")
```

Comment:
Same comment as above. A bit of cleaning up ^^

Reply:
done